internal/artifacts/module_arch.go

```diff
@@ -382,6 +341,12 @@ func packModule(moduleLoc dir.IModule, module *mta.Module, moduleName, platform,
 		return errors.Wrapf(err, packFailedOnTargetArtifactMsg, moduleName)
 	}
 
+	conflictingModule, ok := buildResults[targetArtifact]
+	if ok {
+		return fmt.Errorf(multiBuildWithPathsConflictMsg, conflictingModule, module.Name, moduleLoc.GetTargetModuleDir(module.Name), filepath.Base(targetArtifact))
```

Comment:
Instead of `moduleLoc.GetTargetModuleDir(module.Name)` you could take the folder from `targetArtifact` (like you take the file name from it)

Reply:
done